editor/editor_data.cpp

```diff
@@ -719,6 +719,18 @@ bool EditorData::check_and_update_scene(int p_idx) {
 				new_selection.push_back(new_node);
 			}
 		}
+		if (pscene.is_valid()) {
+			//Apply Overrides when instanced scene gets updated
+			Vector<SceneState::NodeData> o = pscene->get_state()->get_overrides();
+			for (int i = 0; i < o.size(); i++) {
+				Node *e = new_scene->get_node_or_null(pscene->get_state()->get_overrides_path(i, false));
```

Comment:
You used `get_node_or_null()`, but never check if `e` is not null. This code can crash if the node with overrides was removed in the instance.
If `e` is null you could print a warning and skip.